### PR TITLE
Fix update arguments

### PR DESCRIPTION
**Pull request by dhcole**
Quick fix for undefined arguments. cc @DavidEBest

### CODE REVIEW

**Comment by ultrasaurus** [comment]
reviewed by @DavidEBest via alternate communication, I'll wait a bit to see if @polastre or anyone else has any feedback before merging.

@dhcole can you write up a little about what breaks because of this for me to include in midas release notes?

**Comment by dhcole** [comment]
Sure, this was preventing the password reset process from completing, so users couldn't actually reset their password by clicking the reset link. Now, you can reset your password and be taken back to the login screen to log in with your new password.

**Comment by ultrasaurus** [comment]
I'm probably missing something basic, but looks like https://github.com/dhcole/midas/commit/9a7268c1389d1273ed9328d4bfcbd46eacc47b53 is doing an update not a delete...  but the code looks like it only happens on destroy

**Comment by dhcole** [comment]
That's fixing a separate issue: that in midas, the password attempts count is not reset when a user successfully resets their password.

This issue occurs first, when the [reset token is destroyed](https://github.com/dhcole/midas/commit/9a7268c1389d1273ed9328d4bfcbd46eacc47b53#diff-7fe23c833d8e6d7128c87d16967ef450R210). Currently it fails to run `.destroy()` because these arguments are undefined, thus blocking the reset of the reset process from continuing. Two separate issues; both block the Midas password reset process.

File: lib/adapter.js [hunk 1]
```diff
@@ -916,7 +916,7 @@ module.exports = (function() {
         try {
           // build a softdestroy query if setting is enabled
           if (softDelete === true) {
-            query = sequel.update(currentTable, queryObject, { deletedAt: 'NOW()' });
+            query = sequel.update(table, options, { deletedAt: 'NOW()' });
             console.log(query.query);
```
Comment (polastre) [inline lib/adapter.js]: This was one of my debugging statements that should probably be removed, FYI.

**Comment by polastre** [comment]
I don't know how one path of the if uses `table` and `options` and the other uses `currentTable` and `queryObject`.  Not a fan of how this new branch was created that loses the history of the other branches.

**Comment by ultrasaurus** [comment]
@dhcole pls remove the console.log while you've got this open. Then I will merge along with the midas open pull requests and update midas-dev.

**Comment by ultrasaurus** [comment]
@polastre mentioned that unit test should have failed before this commit:  https://github.com/18F/midas/issues/529 for setting up travis on this repo

@dhcole Please confirm that unit tests are passing locally now, since we can't rely on automatic confirmation

**Comment by dhcole** [comment]
Tests pass locally with #3 merged.

screen shot 2015-01-29 at 11 15 02 am

**Pull request merged**